packages/react/src/blocks/forms/TextArea.tsx

```diff
@@ -17,6 +18,7 @@ class TextAreaComponent extends React.Component<FormTextAreaProps> {
         name={this.props.name}
         value={this.props.value}
         defaultValue={this.props.defaultValue}
+        {if (this.props.required) 'required'}
```

Comment:
Are you sure that syntax works? Don't you mean something like:
```suggestion
        required={!!this.props.required}
```
?

Reply:
Another way of doing this
```suggestion
        {...( this.props.required ? { required: 'required' } : {} )}
```